packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -42,9 +42,6 @@ export interface TreeItemProps extends UIComponentProps, ChildrenComponentProps
   /** Only allow one subtree to be open at a time. */
   exclusive?: boolean
 
-  /** Initial open value. */
```

Comment:
Why exactly did we removed this props? I can see that the component was not even extending AutoControlled component, so maybe the changes are correct, but just to make sure.

Reply:
This change is not actually necessary for this PR to work. @layershifter can you please elaborate on this? I think we discovered these props don't belong to  `TreeItem` when looking at the implementation.

Reply:
Yep, `TreeItem` is not extends `AutoControlledComponent`, that's why we decided to remove this cruft